src/server.c

```diff
@@ -1104,7 +1104,19 @@ struct redisCommand redisCommandTable[] = {
 
     {"failover",failoverCommand,-1,
      "admin no-script ok-stale",
-     0,NULL,0,0,0,0,0,0}
+     0,NULL,0,0,0,0,0,0},
+
+    {"publishlocal",publishLocalCommand,3,
+            "pub-sub ok-loading ok-stale fast may-replicate",
+            0,NULL,1,1,1,0,0,0},
+
+    {"subscribelocal",subscribeLocalCommand,-2,
+            "pub-sub no-script ok-loading ok-stale",
+            0,NULL,1,-1,1,0,0,0},
```

Comment:
Isn't this information intended (among other things) for cluster clients to select node by key slot using COMMAND and COMMAND GETKEYS? I think we get stuff for free here if we regard the channels as keys.

Reply:
It's also used for client tracking among other things. I agree with would be a shortcut to getting some support, but I think we shouldn't consider them as keys.

Reply:
Thanks for the heads up, I've handled the scenario(s). Still using the path used for keys for local channels slot mapping.

Reply:
@zuiderkwast I still don't think we should be treating them as keys. pubsub local will have to be implement specifically for clients, since it doesn't operate on keys like other commands. I may be over hesitant here to conflate the two.